src/services/dashboards/modules/DashboardCloneModal.vue

```diff
@@ -158,14 +159,28 @@ export default defineComponent<Props>({
                     return '';
                 });
             }),
+            layouts: computed<DashboardLayoutWidgetInfo[]|DashboardLayoutWidgetInfo[][]>(() => {
+                if (props.dashboard?.layouts) return props.dashboard?.layouts;
+                if ((props.dashboard as unknown as DashboardDetailInfoStoreState)?.dashboardWidgetInfoList) {
+                    return (props.dashboard as unknown as DashboardDetailInfoStoreState)?.dashboardWidgetInfoList;
+                }
+                return [];
+            }),
+            variablesSchema: computed<DashboardVariablesSchema>(() => {
+                if (props.dashboard?.variables_schema) return props.dashboard?.variables_schema;
+                if ((props.dashboard as unknown as DashboardDetailInfoStoreState)?.variablesSchema) {
+                    return (props.dashboard as unknown as DashboardDetailInfoStoreState)?.variablesSchema;
+                }
+                return { properties: {}, order: [] };
+            }),
```

Comment:
suggestion: I think it would be better if an explanation is included in the comments. T^T

Reply:
If we can use ts-ignore😅
```suggestion
            layouts: computed<DashboardLayoutWidgetInfo[]|DashboardLayoutWidgetInfo[][]>(() => {
                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                // @ts-ignore
                return props.dashboard?.layouts ?? props.dashboard?.dashboardWidgetInfoList ?? [];
            }),
            variablesSchema: computed<DashboardVariablesSchema>(() => {
                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                // @ts-ignore
                return props.dashboard?.variables_schema ?? props.dashboard?.variablesSchema ?? { properties: {}, order: [] }
            }),
```

Reply:
Oh...